mapbox/libjava-services/src/test/java/com/mapbox/services/api/directions/v5/MapboxDirectionsTest.java

```diff
@@ -168,27 +171,30 @@ public void noProfileProvided() throws ServicesException {
   }
 
   @Test
-  public void requiredCoordinates() throws ServicesException {
+  public void checksCoordinatesAreRequired() throws ServicesException {
     thrown.expect(ServicesException.class);
     thrown.expectMessage(startsWith("You should provide at least two coordinates (from/to)"));
+
     new MapboxDirections.Builder().setAccessToken("pk.XXX").setProfile(DirectionsCriteria.PROFILE_DRIVING).build();
   }
 
   @Test
-  public void testSanity() throws ServicesException, IOException {
+  public void checksSanity() throws ServicesException, IOException {
     MapboxDirections client = new MapboxDirections.Builder()
       .setAccessToken("pk.XXX")
       .setCoordinates(positions)
       .setProfile(DirectionsCriteria.PROFILE_DRIVING)
       .setBaseUrl(mockUrl.toString())
       .build();
+
     Response<DirectionsResponse> response = client.executeCall();
-    assertEquals(response.code(), 200);
-    assertEquals(response.body().getCode(), DirectionsCriteria.RESPONSE_OK);
+
+    assertEquals(200, response.code());
+    assertEquals("Ok", response.body().getCode());
```

Comment:
@zugaldia it's a technique what I learnt from the book [Working Effectively With Unit Tests](https://leanpub.com/wewut). I really like the techniques described on it in favor of DAMP (Descriptive And Maintainable Procedures). Mainly because they increase maintainability and readability of tests.
In this concrete case, why I prefer _expecting literals_?

- The main advantage of expecting literals is that you only need to focus on the actual value of the assertion, increasing readability and traceability.
If neither your expected nor actual values are literals you’re forced to determine the value of both when a test fails. Conversely, if your expected value is a literal, only the actual value will need to be traced when a test is failing.
- Literals will reflect the examples from the business.